include/download.mk

```diff
@@ -301,6 +302,7 @@ define Download/Defaults
   MIRROR_MD5SUM:=x
   VERSION:=
   OPTS:=
+  SUBMODULES:=
```

Comment:
this changes the default, so I'd prefer if this would then be default to `skip`.

Reply:
The current behavior is to check out all sub modules, which is preserved as the default.